### DISCUSSION TITLE
Exporting from dashboard to Google Sheet

### DISCUSSION THREAD

**Discussion by abadger13** [merchantprotocol/theadslab#49] [User Guide Q&A]
Hi Jonathon,
I can see leads and data in the dashboard, but I can't export them. 
I set up export via integration to Google Sheet and when the sheet was generated I clicked the link in the message at the top of the page, and the page said I was denied access. 
Another time I tried and got an error from Matomo.
I'm logged in to the right Google account. How do I grab the data? Thanks.
![Screen Shot 2022-05-11 at 12 21 19 PM](https://user-images.githubusercontent.com/43648289/168122345-40336711-afb9-4c60-bd10-55ec622fab0a.png)
![Screen Shot 2022-05-11 at 12 21 28 PM](https://user-images.githubusercontent.com/43648289/168122351-4b492634-a9d5-403f-ab49-fdcfd9947324.png)
![Screen Shot 2022-05-12 at 8 39 48 AM](https://user-images.githubusercontent.com/43648289/168122394-e12db4a9-077a-45b6-bd36-a0a61404390e.png)

**Comment by jonathonbyrdziak** [comment]
This should be fixed now. go ahead and save the integration again with your email in the share field.

**Comment by abadger13** [comment]
Sorry to report. All tabs empty. 
I rebuilt the integration (properly. With email). 
All tabs empty, except for headers.
I tested this in a site that's had over 30k site visitors since script installed May 3.

**Comment by goodtreeagency** [comment]
I'm still unable to access the Google Sheet. Is there a solution for this yet?
